modal/_utils/time_utils.py

```diff
@@ -3,13 +3,17 @@
 from typing import Optional
 
 
-def timestamp_to_local(ts: float, isotz: bool = True) -> Optional[str]:
+def timestamp_to_localized_dt(ts: float) -> datetime:
```

Comment:
I kinda think it's preferable to avoid making the return type Optional and forcing users to handle that possibility. Though there is some risk that eventually someone is looking at 1970-01-01 if we start reading dates from the server that haven't always existed.